projecteuler/helpers/number_checks.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+def is_palindromic_number(number):
+    if not isinstance(number, int):
```

Comment:
arguably this function could just be `is_palindrome` and accept any string

Reply:
Done that now, because realised didn't need the initial check too so it's smaller now.